Add remote working tips to communication activities

clausmullie opened the pull request:
-----
[View rendered contributor-guides/for-staff.md](https://github.com/publiccodenet/about/blob/clausmullie-patch-2/contributor-guides/for-staff.md)

clausmullie commented:
Resolves #874

Ainali commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+
+## 'Closing the loop' on open-ended requests.
+
+We identified a potentially harmful patterns around open requests not being 'closed'. 
```
```suggestion
We identified a potentially harmful pattern around open requests not being 'closed'. 
```

Ainali commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+
+This can be demotivating for the initiator (feeling of shouting into the void, not being taken seriously, valued or respected, and feeling like doing meaningless work), expensive for the organisation (many people reading, thinking about it, closing, opening it later again, without coming to a conclusion), and a cycle of the item resurfacing without resolution.
+
+While this loop is easily closed in a collocated working environment (reach over and say hey), a remote environment makes this more tricky.
```
```suggestion
While this loop is easily closed in a co-located working environment (reach over and say hey), a remote environment makes this more tricky.
```

Ainali commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+1) Don't deprioritise with silence, but actively say you can't/won't prioritise something
+2) Set a deadline for responses, and signal what you will do after that deadline (move ahead, or stop working on it if no other interest,...) 
+3) Signal clearly if/when this is a blocker to your work that prevents you from moving on
+4) Where possible ask individuals for input over an undefined group 
```
```suggestion
4) Where possible, ask individuals for input over an undefined group 
```

Ainali commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+While this loop is easily closed in a collocated working environment (reach over and say hey), a remote environment makes this more tricky.
+
+We identified several ways to mitigate this by trying to get a culture around:
+1) Don't deprioritise with silence, but actively say you can't/won't prioritise something
```
```suggestion
1) Don't deprioritize with silence, but actively say you can't/won't prioritize something
```

Ainali commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+3) Signal clearly if/when this is a blocker to your work that prevents you from moving on
+4) Where possible ask individuals for input over an undefined group 
+5) Where possible, provide clear context, aims and expectations around the request
+6) When you take on an open request, try signal whether it is something you will prioritise, or something you will do *if* you have time and space
```
```suggestion
6) When you take on an open request, try signaling whether it is something you will prioritize, or something you will do *if* you have time and space
```

Ainali commented:
Really valuable tips in here Claus! I believe I spotted a few typos, but otherwise it's good to go!

clausmullie commented:
Awesome, thanks @Ainali - eyes of an eagle! :eyes: :eagle:

ElenaFdR commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+
+As we aim to work remote-first, and a lot of our work with codebase communities happens remotely, here are some tips on communication in a remote-first environment.
+
+## Questioning video-call as default. 
```
```suggestion
## Questioning video-call as default
```

ElenaFdR commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+
+This is definitely down to individual preferences, but video calls may be more taxing on the eyes and require more points to pay attention to. 
+
+Some people on the team found that when having 1-on-1 check-in calls or chats, doing it over a phone call while walking around outside worked just as well (or better) and was less taxing/tiring. 
```
```suggestion
Some people on the team found that when having 1-on-1 check-in calls or chats, doing it over a phone call while walking around outside worked just as well (or better) and was less tiring. 
```

ElenaFdR commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+
+Some people on the team found that when having 1-on-1 check-in calls or chats, doing it over a phone call while walking around outside worked just as well (or better) and was less taxing/tiring. 
+
+## 'Closing the loop' on open-ended requests.
```
```suggestion
## 'Closing the loop' on open-ended requests
```

ElenaFdR commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+
+So for example, someone sends an open ended request for input or help or review, but doesn't hear back from anyone.
+
+This can be demotivating for the initiator (feeling of shouting into the void, not being taken seriously, valued or respected, and feeling like doing meaningless work), expensive for the organisation (many people reading, thinking about it, closing, opening it later again, without coming to a conclusion), and a cycle of the item resurfacing without resolution.
```
```suggestion
This can be demotivating for the initiator (feeling of shouting into the void, not being taken seriously, valued or respected, and feeling like doing meaningless work), expensive for the organisation (many people reading, thinking about it, closing, opening it later again, without coming to a conclusion), and lead to a cycle of the item resurfacing without resolution.
```

ElenaFdR commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+
+While this loop is easily closed in a co-located working environment (reach over and say hey), a remote environment makes this more tricky.
+
+We identified several ways to mitigate this by trying to get a culture around:
```
Suggest making this a bulleted, not numbered list, since these are unordered alternatives rather than sequential steps.

ElenaFdR replied:
Also, this list uses 'you' for both the initiator and a respondent. Suggest either identifying roles better, or splitting into advice for initiators and advice for respondents.

clausmullie replied:
Done! Thanks for the helpful advice.

ElenaFdR commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+3) Signal clearly if/when this is a blocker to your work that prevents you from moving on
+4) Where possible, ask individuals for input over an undefined group 
+5) Where possible, provide clear context, aims and expectations around the request
+6) When you take on an open request, try signaling whether it is something you will prioritize, or something you will do *if* you have time and space
```
```suggestion
6) When you take on an open request, try signaling whether it is something you will prioritize, or something you will only do *if* you have time and space
```

ElenaFdR reviewed (commented):
This looks super helpful! There's a disconnect between the PR title and the files you've changed, none of which are the contributing guide for staff. I suggest adding a link to this from the contributing guide for staff.

Ainali commented on activities/communication/communication-remote-first.md:
```diff
@@ -0,0 +1,29 @@
+---
+type: Resource
+---
+
```
Add a top level heading (with one #) with the title of the page.

clausmullie replied:
Done!

clausmullie commented:
@ElenaFdR @Ainali  I believe I have addressed all your feedback. Thanks again for your input!

I've changed the title of this pull request. While first thought this could live in the contributing guide, I'm not sure this is a priority for someone in their first two-three weeks. So I propose putting it in communication activities and leaving it unlinked.

Ainali reviewed (approved):
Great, good to go!

bvhme commented:
This is great! Thanks @clausmullie
